framework/source/class/qx/ui/basic/Image.js

```diff
@@ -587,6 +587,14 @@ qx.Class.define("qx.ui.basic.Image",
         {
           var pixel = "px";
           var styles = {};
+          
+          //inherit styles from current element
+          var currentStyles = currentContentElement.getAllStyles();
+          if( (currentStyles!==null) && (currentStyles!==undefined) ) {
```

Comment:
no not exactly ... it would also be false if `currentStyles` is 0 or if it is an empty string ... but both of these cases would not be good news either here